### PR TITLE
Ingk 1031 virtual drive is not compatible with old disturbance and multiaxis

### PR DESCRIPTION
**Pull request by lucia-teruel**
### Description

Some dictionary products with old disturbance are not compatible with virtual drive.

Fixes #INGK-1031

### Type of change

- Disable Virtual Monitoring and Disturbance for Virtual Drives that have old disturbance.

### Tests
- [x] Add new unit tests if it applies.
- [ ] Run tests.

### Documentation
- [ ] Update docstrings of every function, method or class that change.
- [ ] Build documentation locally to verify changes.
- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting and linting
- [x] Use the ruff package to format the code: `ruff format ingenialink tests virtual_drive`.
- [x] Use the ruff package to lint the code: `ruff check ingenialink tests virtual_drive`.

### Others
- [ ] Set fix version field in the Jira issue.

### CODE REVIEW

File: tests/virtual/test_virtual_network.py [hunk 1]
```diff
@@ -56,3 +57,10 @@ def test_connect_virtual_custom_dictionaries(virtual_drive_custom_dict, read_con
                     value = int(value)
                 servo.write(reg_key, value)
                 assert pytest.approx(value) == server.get_value_by_id(1, reg_key)
+
+
+@pytest.mark.no_connection
+def test_connect_to_virtual_drive_old_disturbance(virtual_drive_custom_dict):
+    dictionary = os.path.join(TESTS_RESOURCES_FOLDER, "ethercat/test_dict_ethercat_old_dist.xdf")
+    server, net, servo = virtual_drive_custom_dict(dictionary)
+    assert servo is not None and net is not None
```
Comment (polfeliu) [inline tests/virtual/test_virtual_network.py:66]: I would verify that _monitoring and _disturbance are None. Just to be sure that the dict used will always trigger this edge case

Comment (lucia-teruel) [reply tests/virtual/test_virtual_network.py:66]: I can add that verification too, but then this test will be exclusive for virtual drives with old_disturbance with this version of the code that disables Virtual Disturbance and Monitoring. Once Virtual Disturbance and Monitoring are allowed for old_disurbance, the test will have to be modified.

Comment (MartinALL91) [reply tests/virtual/test_virtual_network.py:66]: I think you should check that monitoring and disturbance are disabled.

Comment (lucia-teruel) [reply tests/virtual/test_virtual_network.py:66]: Additional check has been added

**Review by MartinALL91 — APPROVED** [review]

File: tests/virtual/test_virtual_network.py (hunk 1 above)
Comment (polfeliu) [reply tests/virtual/test_virtual_network.py:66]: > I can add that verification too, but then this test will be exclusive for virtual drives with old_disturbance with this version of the code that disables Virtual Disturbance and Monitoring. Once Virtual Disturbance and Monitoring are allowed for old_disurbance, the test will have to be modified.

Having to modify tests for new features is expected. No worries

**Review by polfeliu — APPROVED** [review]
